modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/MajorityPeerSelect.scala

```diff
@@ -103,7 +102,7 @@ object MajorityPeerSelect {
         .map(_.take(Math.max(sampleSize, minSampleSize)))
     }
 
-    def getSnapshotHashByPeer(peer: Peer, ordinal: SnapshotOrdinal): F[(Peer, Hash)] =
-      snapshotClient.get(ordinal).run(peer).flatMap(_.toHashed).map(snapshot => (peer, snapshot.hash))
+    def getSnapshotHashByPeer(peer: Peer, ordinal: SnapshotOrdinal): F[Option[(Peer, Hash)]] =
+      snapshotClient.getHash(ordinal).run(peer).map(_.map((peer, _)))
```

Comment:
I see you don't use `getHashFull` method. Why is it required then?

Reply:
I added `getHashFull` because there was `get` and `getFull` for `SnapshotOrdinal`. It has been removed.